formatTest/unit_tests/expected_output/wrappingTest.re

```diff
@@ -2466,169 +2444,136 @@ let produceRecord /*commentBeforeArg*/ x => {
   occupation: "programmer"
 };
 
-let produceRecord x =>
-  /*commentAfterArg*/
-  {
-    name: "hello",
-    age: 20,
-    occupation: "programmer"
-  };
+let produceRecord x /*commentAfterArg*/ => {
+  name: "hello",
+  age: 20,
+  occupation: "programmer"
+};
 
-let myPolyFuncCommentBeforeColon:
+let myPolyFuncCommentBeforeColon /*beforeColon */:
   'a .
-  /*beforeColon */
   'a => 'a
  =
   fun o => o;
 
-let myPolyFuncCommentAfterColon:
-  'a .
+let myPolyFuncCommentAfterColon: 'a .'a => 'a =
   /*afterColon */
-  'a => 'a
- =
   fun o => o;
 
 let myPolyFuncCommentBeforeArrow:
   'a .
-  'a => /*beforeArrow */  'a
+  'a /*beforeArrow */ => 'a
```

Comment:
This is really that "perfect attachment" logic (Step 2) in the algorithm working its magic.